### ISSUE TITLE
Remove restriction to only show in DEBUG mode

### ISSUE THREAD

**Issue by 1c3f0x84** [ActuallyTaylor/HydrogenReporter#7]
Can you please remove the restriction to only show the DebugView only in DEBUG configuration.

**Comment by ActuallyTaylor** [comment]
What would be the benefit of removing the DEBUG mode flags when the debug level is selected? Logs tagged with higher levels will appear in non-debug modes. You can also change your default logging level by setting a log config.

**Comment by 1c3f0x84** [comment]
No, i want to use the debug view in our inhouse app, but i can't to that, because of this DebugView:
```
var body: some View {
        if Config.isDebug && showingSelf {
            internalBody
        } else {
            EmptyView()
        }
    }
```
and in Config
```
    public static var isDebug: Bool {
        #if DEBUG
        return true
        #else
        return false
        #endif
    }
```
The DebugView is only visible in debug build configuration. 

I only want to show the DebugView in Release Build.

**Comment by ActuallyTaylor** [comment]
Ahh I get what you are looking for. This is something that I can get together this weekend. If you can get it done earlier it would be great if you can make a PR to solve the problem.
